ding/torch_utils/network/res_block.py

```diff
@@ -148,3 +148,21 @@ def forward(self, x: torch.Tensor) -> torch.Tensor:
         if self.dropout is not None:
             x = self.dropout(x)
         return x
+
+
+class TemporalSpatialResBlock(nn.Module):
```

Comment:
add usage comments for this class, add typing lint

Reply:
fixed